src/app/store/slices/plan/index.ts

```diff
@@ -189,23 +197,43 @@ export const planSelectors = {
     return currentPlan !== null && currentPlan.isLifetime;
   },
   planLimitToShow: (state: RootState): number => {
+    const { selectedWorkspace } = state.workspaces;
+    if (selectedWorkspace) {
+      const businessPlanLimit = state.plan.businessPlan
+        ? state.plan.businessPlan.storageLimit * state.plan.businessPlan.amountOfSeats
```

Comment:
there is no field that reaches the storage limit of the plan? this will be wrong at some point, as each user may have a different storage.